packages/terra-functional-testing/src/services/wdio-selenium-docker-service.js

```diff
@@ -17,8 +17,9 @@ class SeleniumDockerService {
    * @param {Object} config - The object containing the wdio configuration and options defined in the terra-cli test runner.
    */
   constructor(_options, _capabilities, config = {}) {
-    const { launcherOptions } = config;
-    const { disableSeleniumService, keepAliveSeleniumDockerService, version } = launcherOptions || {};
+    const { launcherOptions, serviceOptions } = config;
+    const { disableSeleniumService, keepAliveSeleniumDockerService } = launcherOptions || {};
+    const { version } = serviceOptions || {};
```

Comment:
Since this wasn't working as expected on the 1.0.0 release we could rename this option if we wanted to.

Reply:
We currently [read the selector](https://github.com/cerner/terra-toolkit/blob/28f4e2d9c0339b01db3a70990f66e187e56835c1/packages/terra-functional-testing/src/services/wdio-terra-service.js#L27) override from [serviceOptions](https://engineering.cerner.com/terra-ui/dev_tools/cerner/terra-functional-testing/wdio-services/terra-service#selector) so renaming it would be non-passive. Do consumers ever need to set the version? If so, should we add some documentation to show this is where to specify the version?

Reply:
Are you referring to renaming `serviceOptions` or `version`. My comment was regarding the name of `version`.

> Do consumers ever need to set the version?

`version` was added so that consumers could periodically update and test against newer selenium docker images. I don't think anyone is doing this today because we rarely update the internal grid, but as we start to make updates more frequently I think this option will become more adopted for testing. 

version is documented [here](https://engineering.cerner.com/terra-ui/dev_tools/cerner/terra-functional-testing/wdio-services/selenium-docker-service#version), but it's out of date now that we no longer accept constructor options. I'll add a note to the documentation update issue

Reply:
Sounds good. I thought you were referring to renaming `serviceOptions` so ignore my comment.

Reply:
Since `version` is no longer provided directly as a service option and is now defined at the flat config level, maybe it makes sense to give it a more descriptive name so that we know what this version refers to. (e.g. `seleniumDockerVersion`, `seleniumDockerImageVersion`, etc).

Reply:
Whoops, I think I merged at the same time your comment came through. There's still time to update it before the release if we want to change it